### PR TITLE
Convert ./components/admin_console/user_grid/user_grid_name.tsx to a function component #24740

### PR DESCRIPTION
**Pull request by sonu27**
#### Summary
This PR converts ./components/admin_console/user_grid/user_grid_name.tsx to a functional component.

#### Ticket Link
Fixes https://github.com/mattermost/mattermost/issues/24740

#### Release Note

```release-note
Converted ./components/admin_console/user_grid/user_grid_name.tsx to a function component
```

### CODE REVIEW

**Review by jespino — APPROVED** [review]

File: webapp/channels/src/components/admin_console/user_grid/user_grid_name.tsx [hunk 1]
```diff
@@ -13,27 +13,27 @@ type Props = {
...
-                    </span>
-                </div>
+const UserGridName = ({
+    user,
+}: Props) => {
+    return (
+        <div className='UserGrid_nameRow'>
+            <ProfilePicture
+                src={Client4.getProfilePictureUrl(user.id, user.last_picture_update)}
+                status={status}
```
Comment (larkox) [inline webapp/channels/src/components/admin_console/user_grid/user_grid_name.tsx:23]: This seems an issue on the original file. Do you mind making the change?
```suggestion
                status={user.status}
```

**Review by larkox — APPROVED** [review]
Awesome, thanks!

**Comment by larkox** [comment]
@lindalumitchell Tested myself on the test server. I checked the team and channel member grids (System Console -> Teams -> Team and System Console -> Channels -> Channel) and they seem to work fine without any problem

**Review by lindalumitchell — APPROVED** [review]
Thanks! Smoke tests you did sound good. e2e failures were unrelated to this PR. I also poked around on the test server and found no issues. LGTM!

**Comment by jespino** [comment]
Merged! Thanks @sonu27! Good to see you around :)

**Comment by sonu27** [comment]
@jespino cheers man. Definitely will :)

**Pull request merged**